### ISSUE TITLE
Table: Nested table col1 wraps unnecessarily

### ISSUE THREAD

**Issue by raineorshine** [cybersemics/em#1900]
## Steps to Reproduce

```
- A
  - =view
    - Table
  - B
    - C
      - =view
        - Table
      - Alpha
        - D
```

Navigate to `Alpha`

## Current Behavior

Col1 wraps after "Alph".

Note that this only occurs after a refresh. If the cursor moves away and then returns to `Alpha`, it does not wrap.

<img width="357" alt="image" src="https://github.com/cybersemics/em/assets/750276/5d035084-1e94-4644-9def-9ccc1a1f6569">

## Expected Behavior

Col1 should not wrap.

**Comment by raineorshine** [comment]
May be related to #1795.

**Comment by raineorshine** [comment]
Fixed in #2412
